spk/sabnzbd/src/wizard/install_uifile_fre

```diff
@@ -1,24 +0,0 @@
-[{
-    "step_title": "Configuration de base",
```

Comment:
For which reason do you discard french translations ? I may provide it.

Reply:
Sure, what should the texts be? 
For install and upgrade.